erpnext/selling/doctype/party_specific_item/party_specific_item.py

```
@@ -17,7 +17,7 @@ class PartySpecificItem(Document):
 
 		based_on_value: DF.DynamicLink
 		party: DF.DynamicLink
-		party_type: DF.Literal["Customer", "Supplier"]
+		party_type: DF.Literal["Customer", "Customer Group", "Supplier", "Supplier Group"]
```

Ruff (0.15.12)

[error] Undefined name `Customer`

(F821)